Clappr.xcodeproj/project.pbxproj

```diff
@@ -276,14 +285,32 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		321781A91FED90CC00EDD678 /* Clappr-tvOS */ = {
+			isa = PBXGroup;
+			children = (
+			);
+			path = "Clappr-tvOS";
+			sourceTree = "<group>";
+		};
+		321781B01FED945400EDD678 /* Sources */ = {
+			isa = PBXGroup;
+			children = (
+				96D362BD1D41339400CCB866 /* Info.plist */,
+				96D362881D41339400CCB866 /* Clappr.h */,
+			);
+			path = Sources;
+			sourceTree = "<group>";
+		};
 		607FACC71AFB9204008FA782 = {
 			isa = PBXGroup;
 			children = (
 				607FACF51AFB993E008FA782 /* Podspec Metadata */,
 				96D362041D41156A00CCB866 /* Example */,
 				607FACE81AFB9204008FA782 /* Tests */,
+				321781B01FED945400EDD678 /* Sources */,
 				96D3627D1D41337600CCB866 /* Clappr */,
 				9E692E001F9F898F006510BE /* Clappr_UITests */,
+				321781A91FED90CC00EDD678 /* Clappr-tvOS */,
```

Comment:
@igoriols as this folder is empty, it wasn't really created (it is red in Xcode). I suggest you to really create the folder and maybe use a .gitkeep file